### PR TITLE
Use ANSI dim instead of hardcoding colour

### PR DESCRIPTION
**Pull request by kse**
Instead of hardcoding the colour for file and types, use the dim escape code to let the terminal decide the colour.
Specifically this makes it more readable if your terminal happens to be slightly gray.

Also includes some `gofmt` edits of package comment.

### CODE REVIEW

**Review by matryer — APPROVED** [review]
This seems great - can you share a screenshot of what it looks like?

**Comment by flowchartsman** [comment]
image
Kinda mixed for me, but adaptive is certainly better than not. I'd be inclined to take it a step further and integrate  the [xterm Operating System Commands](https://invisible-island.net/xterm/ctlseqs/ctlseqs.html#h3-Operating-System-Commands) `OSC 10 ; ? BEL` and/or `OSC 11 ; ? BEL` for color selection. Could be part of a larger effort since, AFAICT Windows terminal colors aren't handled (but I haven't tested)

**Comment by matryer** [comment]
Thanks @flowchartsman

**Comment by kse** [comment]
Thanks for approving and sorry for not getting back to you before flowchartsman. :)
Kids, sparetime and all.

**Comment by matryer** [comment]
thanks @kse - what do you think about [his previous comment](https://github.com/matryer/is/pull/50#issuecomment-1444585649)?

**Comment by kse** [comment]
I admittedly know nothing about the `xterm` OS commands.
The only color set right now is (dark) red. I think this color is legible enough on both a black-ish or white-ish background to not be an issue.
There are of course red-green color blind people, but I think the contrast is likely clear enough in any case.

An alternative to a color is of course **bold**, which is the only proper alternative IMO.

```
echo -e "\u001b[01m Hello world\u001b[00m in bold"
```

Based on the (rather excellent) [wikipedia article](https://en.wikipedia.org/wiki/ANSI_escape_code), the ANSI codes should work on some (modern) Windows Terminal as well (though not Windows Console).
But I suspect most developers would use Terminal or some Linux shell through WSL.

(I only have Linux to test on).

TL;DR

I think this is a good improvement and we don't need to dynamically figure out the colour of the comment.

**Comment by flowchartsman** [comment]
I'm not familiar with Windows these days, but if the ANSI codes work on Windows Terminal, I think that's good enough fore me to cap it here and not to bother with anything fancier. I also agree that bold is a great option, and might be worth considering elsewhere or in place of dim. In any case, probably good as it is unless someone complains. Thanks for the work!
